MycroftAI/selene-backend
# Feature/timezone endpoint

## Description
Add an endpoint to the public device API that returns geolocation data based on a word that might represent city, state or country.

## How to test
Call the endpoint with names of different geological locations and verify the response.

## Contributor license agreement signed?
CLA [Y]
